docs/puzzles/cats.md

```diff
@@ -187,6 +200,51 @@ The CAT1 standard currently includes three example TAILs, though many more are p
   
   There is another consideration to make when you are signing new Delegated TAILs. Once you sign it and publish the signature, it is out there forever. Be careful what permissions you grant because you can never take them back.
   
-  
+## CAT denominations, value, and retirement rules
+
+Some design decisions regarding the granularity and denomination of CATs versus XCH:
+
+* Most Chia wallets choose to display their value in XCH. However, this is a purely cosmetic choice because Chia's blockchain only knows about mojos. One XCH is equal to one trillion (1,000,000,000,000) mojos.
+* One CAT token is equal to 1000 XCH mojos.
```

Comment:
This is I think the crux of the confusion.  I think you should say it like "Just like 1 XCH maps to 1 trillion XCH mojos, a default decision was made in the GUI to map 1 CAT to 1000 CAT mojos" or something like that.

Reply:
Yes I like this idea.